tests/pack/pack_tests.cpp

```diff
@@ -195,11 +195,24 @@ struct TestPack {
     }
   }
 
+  static void test_mixed_prec () {
+    const ekat::Pack<float, PACKN> ones(1.0);
+    const ekat::Pack<double, PACKN> twos(2.0);
+
+    const ekat::Pack<double, PACKN> dones(ones);
+    const ekat::Pack<float, PACKN> ftwos(twos);
+
+    vector_novec for (int i=0; i<PACKN; ++i) {
+      REQUIRE( std::abs(dones[i] - 1.0) < 1e-7 );
```

Comment:
Do you want a lower tolerance for the `double` version to illustrate what's going on here?